stdlib/sys.mli

```diff
@@ -27,7 +27,9 @@ val argv : string array
    given to the program. *)
 
 val executable_name : string
-(** The name of the file containing the executable currently running. *)
+(** The name of the file containing the executable currently running.
+    The path may be relative or absolute depending on the platform and
```

Comment:
Suggested edit:
`This name may be absolute or relative to the current directory, depending on...`